pkg/common/label.go

```diff
@@ -15,6 +15,8 @@ limitations under the License.
 
 package common
 
+import "regexp"
+
 const (
 	// LabelAnnotationPrefix is the prefix of every labels and annotations added by the controller.
 	LabelAnnotationPrefix = "fluid.io/"
```

Comment:
Can we use `common.LabelAnnotationPrefix`  instead of defining a new constant.

Reply:
It is a regex variable, not a constant to match the label.